### PR TITLE
SDD pipeline 1.1.1

### PR DESCRIPTION
**Pull request by akshay-dlpx**
renamed an activity, added services and categories in manifest.json

### CODE REVIEW

File: community templates/Sensitive Data Discovery with Delphix/Sensitive Data Discovery with Delphix.json [hunk 1]
```diff
@@ -1,4709 +1 @@
...
...pe":"Expression"},"SourceRulesetName":{"value":"@pipeline().parameters.SourceRulesetName","type":"Expression"},"TargetRulesetName":{"value":"@pipeline().parameters.TargetRulesetName","type":"Expression"},"RootFolder":{"value":"@pipeline().parameters.fileShareRootFolder","type":"Expression"}}}},{"name":"Get Source Metadata","type":"GetMetadata","dependsOn":[{"activity":"Copy Data to File Format Directory","dependencyConditions":["Succeeded"]}],"policy":{"timeout":"0.12:00:00","retry":0,"retryIntervalInSeconds":30,"secureOutput":false,"secureInput":false},"userProperties":[],"typeProperties":{"dataset":{"referenceName":"fileFormatDataSetPiped","type":"DatasetReference","parameters":{"containerName":{"value":"@pipeline().parameters.ContainerRootFolder","type":"Expression"},"fileName":"*","directoryName":"current-fileformat/fileformat"}},"fieldList":["childItems"],"storeSettings":{"type":"AzureBlobStorageReadSettings","recursive":true,"enablePartitionDiscovery":false},"formatSettings":{"type":"DelimitedTextReadSettings"}}},{"name":"Get Target Metadata","type":"GetMetadata","dependsOn":[{"activity":"Copy Data to File Format Directory","dependencyConditions":["Succeeded"]}],"policy":{"ti
```
Comment (kishlay-delphix) [inline community templates/Sensitive Data Discovery with Delphix/Sensitive Data Discovery with Delphix.json]: Please add formatting

Comment (kishlay-delphix) [inline community templates/Sensitive Data Discovery with Delphix/Sensitive Data Discovery with Delphix.json]: Set `contentVersion` to `1.1.1.0`

File: community templates/Sensitive Data Discovery with Delphix/manifest.json [hunk 2]
```diff
@@ -1,38 +1 @@
...
... y=\"0\" width=\"192\" height=\"90\" fill=\"#F4F4F4\" stroke=\"none\" transform=\"matrix(1, 0, 0, 1, 0, 0)\"/><g transform=\"matrix(1, 0, 0, 1, 0, 0)\"><g transform=\"matrix(1, 0, 0, 1, 0, 0)\"><rect x=\"0\" y=\"0\" width=\"192\" height=\"25\" fill=\"#DCDCDC\" stroke=\"none\" transform=\"matrix(1, 0, 0, 1, 0, 0)\"/><g transform=\"matrix(1, 0, 0, 1, 0, 0)\" clip-path=\"url(#CLIP3971004)\"><text x=\"0\" y=\"10.506005859375001\" style=\"font: 12px Segoe UI\" text-anchor=\"start\" fill=\"#515151\" stroke=\"none\" transform=\"matrix(1, 0, 0, 1, 5, 6.5)\">Get Metadata</text></g><g transform=\"matrix(1, 0, 0, 1, 0, 0)\" clip-path=\"url(#CLIP3971006)\"><g transform=\"matrix(1, 0, 0, 1, 142, -3)\"><path d=\"M 1.8409491661026458,0 L 3.6818983322052947,0 C 4.506119348655733,0 5.52284749830794,1.0167281496522078 5.52284749830794,1.8409491661026458 L 5.52284749830794,3.681898332205292 C 5.52284749830794,4.50611934865573 4.506119348655733,5.522847498307938 3.6818983322052947,5.522847498307938 L 1.8409491661026458,5.522847498307938 C 1.0167281496522078,5.522847498307938 0,4.50611934865573 0,3.681898332205292 L 0,1.8409491661026458 C 0,1.0167281496522078 1.0167281496522078,0 1.8409491661026458,0 z
```
Comment (kishlay-delphix) [inline community templates/Sensitive Data Discovery with Delphix/manifest.json]: Please add formatting for this file as well

**Review by kishlay-delphix — CHANGES_REQUESTED** [review]

File: community templates/Sensitive Data Discovery with Delphix/Sensitive Data Discovery with Delphix.json (hunk 1 above)
Comment (akshay-dlpx) [reply community templates/Sensitive Data Discovery with Delphix/Sensitive Data Discovery with Delphix.json]: done

File: community templates/Sensitive Data Discovery with Delphix/manifest.json (hunk 2 above)
Comment (akshay-dlpx) [reply community templates/Sensitive Data Discovery with Delphix/manifest.json]: done

File: community templates/Sensitive Data Discovery with Delphix/Sensitive Data Discovery with Delphix.json (hunk 1 above)
Comment (akshay-dlpx) [reply community templates/Sensitive Data Discovery with Delphix/Sensitive Data Discovery with Delphix.json]: done

File: community templates/Sensitive Data Discovery with Delphix/manifest.json [hunk 3]
```diff
@@ -26,13 +26,24 @@
             }
         }
     },
-    "author": "Delphix",
+    "author": "kishlay.singh@azureopsdelphix.onmicrosoft.com",
```
Comment (kishlay-delphix) [inline community templates/Sensitive Data Discovery with Delphix/manifest.json]: Can you please change the author to Delphix

**Review by kishlay-delphix — APPROVED** [review]

File: community templates/Sensitive Data Discovery with Delphix/manifest.json (hunk 3 above)
Comment (akshay-dlpx) [reply community templates/Sensitive Data Discovery with Delphix/manifest.json]: corrected the value

**Pull request merged**